includes/polling/os/netonix.inc.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+
```

Comment:
I can remove that, it's just got nothing available at the moment to get things like serial, hardware out.